cinema/urls.py

```diff
@@ -11,13 +11,18 @@
 )
 
 router = routers.DefaultRouter()
-router.register("genres", GenreViewSet)
-router.register("actors", ActorViewSet)
-router.register("cinema_halls", CinemaHallViewSet)
-router.register("movies", MovieViewSet)
-router.register("movie_sessions", MovieSessionViewSet)
-router.register("orders", OrderViewSet)
+router.register("moviesessions", MovieSessionViewSet)
 
-urlpatterns = [path("", include(router.urls))]
+urlpatterns = [
+    path("", include(router.urls)),
+    path("actors/", ActorViewSet.as_view(), name="actor-list"),
+    path("genres/", GenreViewSet.as_view(), name="genre-list"),
+    path("orders/", OrderViewSet.as_view({"get": "list"}), name="order-list"),
+    path("cinema_halls/", CinemaHallViewSet.as_view(), name="cinemahall-list"),
```

Comment:
The `CinemaHallViewSet` should be registered with the router instead of being used directly in the `path` function. This is because it is a viewset and should be accessed through the router's automatically generated URLs.